simtfl/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+This is an experimental simulator for research into a potential [Trailing Finality
+Layer](https://electriccoin.co/blog/the-trailing-finality-layer-a-stepping-stone-to-proof-of-stake-in-zcash/)
```

Comment:
```suggestion
This is an experimental simulator for research into a potential [Trailing Finality
Layer](https://electric-coin-company.github.io/tfl-book/)
```

Let's consolidate the book rendering as the primary "portal", since it's a living doc, whereas the blog post may become outdated. Also, the book has a PR to link back to this project to reinforce their relationship.